cmd/platform/user.go

```diff
@@ -202,7 +201,7 @@ func userDeactivateCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) < 1 {
-		return errors.New("Enter user(s) to deactivate.")
+		return errors.New("Expected at least one argument. See help text for details.")
```

Comment:
Is it common to include a link to help documentation? (I'm not that familiar with the CLI tools).

If so, we could also link to it with, say, `Expected at least one argument. See documentation for details. [link]`

^I'm 0/5, we should do what's standard for CLI tools.